microsoft-edge/progressive-web-apps-chromium/how-to/icon-theme-color.md

```diff
@@ -60,10 +60,224 @@ Each icon in the `icons` array should come with at least the `src` and `sizes` p
+    * 66x66 - app icon 1.5x display scale
+    * 88x88 - app icon 2x display scale
+    * 176x176 - app icon 4x display scale
+    * 71x71 - small tile
+    * 89x89 - small tile 1.25x display scale
+    * 107x107 - small tile 1.5x display scale
+    * 142x142 - small tile 2x display scale
+    * 284x284 - small tile 4x display scale
+    * 150x150 - medium tile
+    * 188x188 - medium tile 1.25x display scale
+    * 225x225 - medium tile 1.5x display scale
+    * 300x300 - medium tile 2x display scale
+    * 600x600 - medium tile 4x display scale
+    * 350x150 - wide tile
+    * 388x188 - wide tile 1.25x display scale
+    * 465x225 - wide tile 1.5x display scale
+    * 620x300 - wide tile 2x display scale
+    * 1240x600 - wide tile 4x display scale
+    * 310x310 - large tile
+    * 388x388 - large tile 1.25x display scale
+    * 465x465 - large tile 1.5x display scale
+    * 620x620 - large tile 2x display scale
+    * 1240x1240 - large tile 4x display scale
+    * 50x50 - store tile
+    * 63x63 - store tile 1.25x display scale
+    * 75x75 - store tile 1.5x display scale
+    * 100x100 - store tile 2x display scale
+    * 200x200 - store tile 4x display scale
+    * 620x300 - splash screen
+    * 775x375 - splash screen 1.25x display scale
+    * 930x450 - splash screen 1.5x display scale
+    * 1240x600 - splash screen 2x display scale
+    * 2480x1200 - splash screen 4x display scale
+    * 16x16 - target size for taskbar, start menu, task manager
+    * 20x20 - target size for taskbar, start menu, task manager
+    * 24x24 - target size for taskbar, start menu, task manager
+    * 30x30 - target size for taskbar, start menu, task manager
+    * 32x32 - target size for taskbar, start menu, task manager
+    * 36x36 - target size for taskbar, start menu, task manager
+    * 40x40 - target size for taskbar, start menu, task manager
+    * 44x44 - target size for taskbar, start menu, task manager
+    * 48x48 - target size for taskbar, start menu, task manager
+    * 60x60 - target size for taskbar, start menu, task manager
+    * 64x64 - target size for taskbar, start menu, task manager
+    * 72x72 - target size for taskbar, start menu, task manager
+    * 80x80 - target size for taskbar, start menu, task manager
+    * 96x96 - target size for taskbar, start menu, task manager
+    * 256x256 - target size for taskbar, start menu, task manager
+In level 4, you supply images for tiles with display scales and target size images for display in various surfaces in Windows, including taskbar, start menu, task manager, ALT+Tab task switcher, and more.
 
 <!-- ====================================================================== -->
+### Image level summary
+
+Consider, for example, how your app icon might show different content depending on the available surface area:
+
+* Small surface: App icon (44x44) in taskbar, showing a small, abbreviated logo: !
+[The small size icon](../media/windows-image-app-icon.png) 
```

Comment:
The `!` character needs to be right next to the `[` character for the image to load. Otherwise, it won't work.
Also, as suggested before, you need the image to be on its own line to become its own block in the rendered docs.

So, your list should be formatted like this instead:

```markdown
* Small surface: App icon (44x44) in taskbar, showing a small, abbreviated logo:

   ![The small size icon](../media/windows-image-app-icon.png)

* Larger surface: Medium tile (150x150) in start menu, showing the full logo of the app:  

   ![The medium size icon.](../media/windows-image-small-tile.png) 

* Wide surface: Wide tile in start menu, showing a more articulated app icon: 

   ![The wide size icon](../media/windows-image-wide-tile.png) 
```

Notice how I've left empty lines between bullet points and images, and also how I've indented the images. Indentation is important to make sure the images actually appear aligned with their parent bullet points.

Reply:
fixed.